src/ts/confirmBtnClick.ts

```diff
@@ -0,0 +1,73 @@
+import { showNotEmptyCart } from "./cart-page/displayCart";
+import {
+  cardNumberInput,
+  cardDateInput,
+  cvvInput,
+  personNameIn,
+  phoneNumberIn,
+  addressIn,
+  emailIn,
+  modalWindowWrap,
+  modalWindow,
+} from "./purchase-window/startPurchaseWindow";
+
+export function confirmBtnClick() {
+  const confirmBtn = document.querySelector<HTMLElement>(".btn__confirm");
+
+  confirmBtn?.addEventListener("click", () => {
+    if (cardNumberInput?.value.length === 0) {
+      const myevent = new Event("change");
+      cardNumberInput.dispatchEvent(myevent);
+    }

```

Comment:
```suggestion
    if (cardNumberInput?.value.length === 0) {
      cardNumberInput.dispatchEvent(new Event("change"));
    }
```
the same for others

Reply:
Fixed by be18f475cf1ff1abaa06c04a4b15eeae859a9782